pkg/api-server/server.go

```diff
@@ -29,6 +32,24 @@ func (a *ApiServer) Address() string {
 	return a.server.Addr
 }
 
+func init() {
+	// turn off escape & character so the link in "next" fields for resources is user friendly
```

Comment:
Does it turn off escaping for all api-server responses?

Reply:
Yes, you cannot differentiate it.

Reply:
Marco decided that we should not escape it for better UX